### ISSUE TITLE
Terminal find doesn't have border around input

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#189646]
Look after https://github.com/microsoft/vscode/pull/179398 is merged in Dark Modern:

![image](https://github.com/microsoft/vscode/assets/2193314/5b9c888d-f1ba-4244-84f7-4f26abb04dab)

**Comment by daviddossett** [comment]
I see that it's currently using `contrastBorder` via https://github.com/microsoft/vscode/blob/cd1af5b0601470fbe9d58abe861485c9fdceee3b/src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css#L34

Changing it to `widget-border` works, but I'm wondering what exactly broke this and why it was using `contrastBorder` in the first place?

**Comment by tisilent** [comment]
![12](https://github.com/microsoft/vscode/assets/48614781/579fd870-bfec-4e08-a76e-4971942533ff)

left : version 81  right : insiders
Maybe it's related to background-color ?

**Comment by tisilent** [comment]
![企业微信截图_16912308653484](https://github.com/microsoft/vscode/assets/48614781/a9a5fca7-111c-4667-877f-a9bc7559a474)

test, 
--vscode-panelInput-border color effect

**Comment by tisilent** [comment]
hi @Tyriar , I don't know if it's some new issues.

**Comment by Tyriar** [comment]
@daviddossett is the find bg color meant to look like that?

**Comment by shannawha** [comment]
I broke it .... Leave it alone please it's not even yours ... It's military on base even they caught you I bet you how much u want to bet they didn't even detect you messing around with their airplane.... Octo'cat

________________________________

**Comment by shannawha** [comment]
Onemore thing .. it's a look alike ... Or next of kin  just be a picture I have that involve my cousin walking around my airplane.

________________________________

**Comment by daviddossett** [comment]
Yes—changes made in https://github.com/microsoft/vscode/pull/189591 to make editor widget and quick input backgrounds lighter to more easily distinguish against the editor background.

**Comment by tisilent** [comment]
Yes, it's normal.Terminal simple find widget just not synchronized with the editor.
